bootstrap/absolute.sh

```diff
@@ -10,18 +10,25 @@
 # Copy githooks to .git/hooks.
 cp $(githooks_path)/* $(absolute_path)/.git/hooks
 
-# Sync third_parties.
-sync_node
-sync_mongodb
-
 # Set path
+if is_windows_platform; then
+  if [ "$1" = "from-cmd" ]; then
```

Comment:
That "if" statement is to export "$(third_party_path)/win-bash" path only user launch absolute in windows cmd. Without this line, bash-shell that already installed in user windows pc, run command in $(third_party_path)/win-bash.